src/Yaapii.Http/Parts/Headers/Authorization.FirstOf.cs

```diff
@@ -38,7 +39,8 @@ public sealed class FirstOf : TextEnvelope
             /// </summary>
             public FirstOf(IDictionary<string, string> input) : base(() =>
                 new FirstOf<string>(
-                    new Authorization.Of(input)
+                    new Authorization.Of(input),
+                    new InvalidOperationException($"Failed to extract first value of '{Authorization.KEY}' header. No values found.")
```

Comment:
I think it would be more understandable to write 'Cannot find 'Authorization.KEY' in headers' - from the outside this is less technical.